src/stories/GardenEditor.stories.js

```diff
@@ -5,34 +5,127 @@
 import { CKEditor } from '@ckeditor/ckeditor5-react';
 import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
+import Code from '@ckeditor/ckeditor5-basic-styles/src/code';
+import CodeBlock from '@ckeditor/ckeditor5-code-block/src/codeblock';
 import Essentials from '@ckeditor/ckeditor5-essentials/src/essentials';
+import Heading from '@ckeditor/ckeditor5-heading/src/heading';
+import HorizontalLine from '@ckeditor/ckeditor5-horizontal-line/src/horizontalline';
+import Indent from '@ckeditor/ckeditor5-indent/src/indent';
+import IndentBlock from '@ckeditor/ckeditor5-indent/src/indentblock';
 import Italic from '@ckeditor/ckeditor5-basic-styles/src/italic';
+import Link from '@ckeditor/ckeditor5-link/src/link';
+import List from '@ckeditor/ckeditor5-list/src/list';
 import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
 import Underline from '@ckeditor/ckeditor5-basic-styles/src/underline';
+import Widget from '@ckeditor/ckeditor5-widget/src/widget';
+import { add } from '@ckeditor/ckeditor5-utils/src/translation-service';
+import { getEnvKeystrokeText } from '@ckeditor/ckeditor5-utils/src/keyboard';
+
+// import GardenEditor from '../gardeneditor';
+import './theme/styles.css';
+import { INITIAL_DATA } from './initialData';
+
+const EDITOR_LOCAL_STORAGE_KEY = 'ckeditor5-demo-1';
 
 export default {
   title: 'Examples'
 };
 
+/**
+ * Add custom tooltip values
+ */
+add('en', {
+  'Bulleted List': `Bulleted list (${getEnvKeystrokeText('CTRL+SHIFT+8')})`,
+  'Numbered List': `Numbered list (${getEnvKeystrokeText('CTRL+SHIFT+7')})`,
+  'Decrease indent': `Decrease indent (${getEnvKeystrokeText('CTRL+[')})`,
+  'Increase indent': `Increase indent (${getEnvKeystrokeText('CTRL+]')})`,
+  'Block quote': `Blockquote (${getEnvKeystrokeText('CTRL+SHIFT+9')})`,
+  Code: `Code (${getEnvKeystrokeText('CTRL+SHIFT+5')})`,
+  'Horizontal line': `Horizontal rule (${getEnvKeystrokeText('CTRL+SHIFT+L')})`
+});
+
 export const Default = () => {
+  const [storedValue] = useState(() => {
+    const storedData = window.localStorage.getItem(EDITOR_LOCAL_STORAGE_KEY);
+
+    return storedData || INITIAL_DATA;
+  });
+
+  const onValueChange = useCallback(value => {
+    window.localStorage.setItem(EDITOR_LOCAL_STORAGE_KEY, value);
+  }, []);
+
   return (
     <CKEditor
       editor={ClassicEditor}
+      data={storedValue}
       config={{
-        plugins: [Essentials, Paragraph, Bold, Italic, Underline],
-        toolbar: ['bold', 'italic', 'underline']
+        plugins: [
+          Essentials,
+          Bold,
+          Italic,
+          Paragraph,
+          Underline,
+          Code,
+          HorizontalLine,
+          Link,
+          Indent,
+          IndentBlock,
+          List,
+          BlockQuote,
+          Heading,
+          CodeBlock,
+          Widget
+        ],
+        toolbar: [
+          'heading',
+          'bold',
+          'italic',
+          'underline',
+          '|',
```

Comment:
Can separators be handled responsively? It feels strange that this appears before the overflow menu when the editor shrinks down to "small mobile" size.

Reply:
Weird, this is being removed as I would expect in mobile for me 
![image](https://user-images.githubusercontent.com/4030377/104218969-5947c980-53f2-11eb-9b4b-50564de87ea1.png)

Reply:
I can see the vertical separator before the overflow button in the screen shot you provided ☝️

Reply:
I think that is a separate `ck-separator` that is injected by the toolbar. It's not controlled by the toolbar config.

Reply:
Not a super high priority, but can this be dealt with via CSS? We should make this thing bend to our styling will wherever possible.

Reply:
It looks like it's appended to the DOM and doesn't include any targetable classes. I'll make a not to take a look at this along with the other toolbar customizations.